MDExtensions/MDNodeExtensions.cs

```diff
@@ -298,6 +347,31 @@ public static object MDServerRpcUnreliable(this Node Instance, string Method, pa
         return Instance.MDRpcUnreliableId(ServerId, Method, Args);
     }
 
+    public static bool Invoke(this Node Instance, String Method, params object[] Parameters)
+    {
+        Type NodeType = Instance.GetType();
+        MethodInfo Info = NodeType.GetMethod(Method, BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
I think this is where FastMember would help as this is a lookup by string. Something we can look into in the future.